AnkiConnect.py

```diff
@@ -89,11 +89,11 @@ def makeStr(data):
 def download(url):
     try:
         resp = web.urlopen(url, timeout=URL_TIMEOUT)
-    except web.URLError:
-        return None
+    except web.URLError as e:
+        raise Exception("A urlError has occoured for url " + url + ". Error messages was: " + e.message)
```

Comment:
Single quotes instead of double quotes please (all throughout this PR)

Reply:
Will do.